public/components/cards/27.html

```diff
@@ -0,0 +1,48 @@
+<a class="block bg-[#fefefe] shadow-xl rounded-xl text-center p-5" href="">
+  <h5
+    class="text-3xl font-bold mb-1 text-transparent bg-clip-text bg-gradient-to-r from-blue-600 via-blue-500 to-blue-600"
```

Comment:
Gradient is not very noticeable and isn't needed, can be `text-blue-600`.

Remove the `mb-1`.

Reply:
Some gradient classes have been leftover.